backend/src/main/java/com/soen341/instagram/controller/RegistrationController.java

```diff
@@ -0,0 +1,11 @@
+package com.soen341.instagram.controller;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
We might use @Controller in the future instead. For now it's O.K. But we'll need to discuss this more in the future